app/controllers/api/base_controller/authentication.rb

```diff
@@ -91,8 +91,9 @@ def api_token_mgr
         Environment.user_token_service.token_mgr('api')
       end
 
-      def auth_user(userid)
-        auth_user_obj = User.lookup_by_identity(userid)
+      def auth_user(user_or_id)
+        auth_user_obj   = user_or_id if user_or_id.kind_of?(User)
+        auth_user_obj ||= User.lookup_by_identity(user_or_id, lookup_scope: :api_includes)
```

Comment:
is it possible to write this as an `if / else`, or `?:`

Reply:
Yeah, I am down with that.  Probably would be much easier to grok.  👍